### DISCUSSION TITLE
Treat document metadata  (in the *.qmd file ) as first party metadata

### DISCUSSION THREAD

**Discussion by mckabue** [quarto-dev/quarto-cli#9855] [Feature Requests]
**/posts/welcome/index.qmd**

```
---
title: "Welcome To My Blog"
date: "2024-05-01"
categories: [news]
author: "martin from welcome"
---

This is the first post in a Quarto blog. Welcome!

![](thumbnail.jpg)

Since this post doesn't specify an explicit `image`, the first image in the post will be used in the listing page of posts.
```

**/posts/_metadata.yml**
```
categories: [tech]
author:
  - name: James from posts
```

With the above setup, I get the following post and listing:

<img width="674" alt="Screenshot 2024-06-03 at 22 21 52" src="https://github.com/quarto-dev/quarto-cli/assets/6072072/aadea43a-4875-40a5-bba2-a0667f2c2bfa">
<img width="939" alt="Screenshot 2024-06-03 at 22 22 14" src="https://github.com/quarto-dev/quarto-cli/assets/6072072/55e9e0c6-ad6c-4995-a3c2-1f11ab6fe0c6">

*****

However, if:

**/posts/welcome/index.qmd**

```
---
title: "Welcome To My Blog"
date: "2024-05-01"
---

This is the first post in a Quarto blog. Welcome!

![](thumbnail.jpg)

Since this post doesn't specify an explicit `image`, the first image in the post will be used in the listing page of posts.
```

**/posts/welcome/_metadata.yml**
```
categories: [news]
author: "martin from welcome"
```

**/posts/_metadata.yml**
```
categories: [tech]
author:
  - name: James from posts
```

I get the following post and listing:

<img width="939" alt="Screenshot 2024-06-03 at 22 25 43" src="https://github.com/quarto-dev/quarto-cli/assets/6072072/cef4bb57-738b-4f39-ac0a-30097b08566c">
<img width="939" alt="Screenshot 2024-06-03 at 22 25 54" src="https://github.com/quarto-dev/quarto-cli/assets/6072072/5dcdb8b7-08aa-40ac-a904-068a3dd487d3">

*****

When the metadata is in the document file, it overwrites all parent metadata such as authors and categories instead of extending it in the post page.

**Comment by cderv** [comment]
It seems the behavior is correct in the listing right ? 

The image you shared shows both authors in there, and this is what you expect ? 
Same for tags ? 

Both are in the listing part, but not in the document

**Comment by mckabue** [comment]
Yes, the behavior is correct on the **listing page.**

However, the behavior is not correct on the **post page**

**Reply by cderv** [reply]
I'll create an issue for this. I think this is not limited to blog and listing, but is happening for any use of `author` in document and elsewhere

**Reply by mcanouil** [reply]
I am pretty sure there was a discussion/issue about this but cannot find it ... (maybe it was only a side comment in a discussion/issue)

**Reply by mcanouil** [reply]
Found it:
- https://github.com/quarto-dev/quarto-cli/discussions/8860

it was another issue were all were merged resulting in duplicates, the fix was: https://github.com/quarto-dev/quarto-cli/commit/478b3df44b6db1e97d9bb25d5ec4dc6b0f33cdb8

**Reply by cderv** [reply]
This is another one right ? Other topic ? 

I opened this one, because I think we have a problem
* https://github.com/quarto-dev/quarto-cli/issues/9864

**Comment by mckabue** [comment]
What about `categories` field?

**Comment by mckabue** [comment]
when _metadata.yml file is used, `categories` are merged in the post page, but when metadata in the document, they are not merged.

**Reply by cderv** [reply]
Same problem I believe. What I describe in the issue is not only for authors.
